set.mm

```diff
@@ -20050,32 +20250,854 @@ requires either a disjoint variable condition ( ~ sb56 ) or a non-freeness
+      UABLDMLZUCDEFUGNZUFBUGDEUHGRST $.
+  $}
+
+  ${
+    $( ` ta ` means ` [ y / x ] ps ` a la ~ dfsb1 . $)
+    dfsb1.s5 $e |- ( ta <-> ( ( x = y -> ps ) /\ E. x ( x = y /\ ps ) ) ) $.
+    sbiedALT.1 $e |- F/ x ph $.
+    sbiedALT.2 $e |- ( ph -> F/ x ch ) $.
+    sbiedALT.3 $e |- ( ph -> ( x = y -> ( ps <-> ch ) ) ) $.
+    $( Alternate version of ~ sbied .  (Contributed by NM, 30-Jun-1994.)
+       (Revised by Mario Carneiro, 4-Oct-2016.)  (Proof shortened by Wolf
+       Lammen, 24-Jun-2018.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    sbiedALT $p |- ( ph -> ( ta <-> ch ) ) $=
+      ( wi weq wa wex biid sbrimALT nfim1 wb com12 pm5.74d sbieALT pm5.74ri
+      bitr3i ) ADCADKEFLZABKZKUDUEMENMZACKZABDUFEFGUFOZHPUEUGUFEFUHACEHIQUDABCA
+      UDBCRJSTUAUCUB $.
+  $}
+
+  ${
+    $( ` th ` means ` [ y / x ] ph ` a la ~ dfsb1 . $)
+    dfsb1.p8 $e |- ( th <-> ( ( x = y -> ph ) /\ E. x ( x = y /\ ph ) ) ) $.
+
+    $( ` ta ` means ` [ y / z ] [ z / x ] ph ` a la ~ dfsb1 . $)
+    dfsb1.sb $e |- ( ta <-> (
+      ( z = y -> ( ( x = z -> ph ) /\ E. x ( x = z /\ ph ) ) ) /\
+      E. z ( z = y /\ ( ( x = z -> ph ) /\ E. x ( x = z /\ ph ) ) ) ) ) $.
+    sbco2ALT.1 $e |- F/ z ph $.
+    $( Alternate version of ~ sbco2 .  (Contributed by NM, 30-Jun-1994.)
+       (Revised by Mario Carneiro, 6-Oct-2016.)  (Proof shortened by Wolf
+       Lammen, 17-Sep-2018.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    sbco2ALT $p |- ( ta <-> th ) $=
+      ( weq wal wb wi wa wex sbequ12ALT biid sbequALT bitr3d sps nfnae nfsb4ALT
+      wn a1i sbiedALT pm2.61i ) FEJZFKZCBLZUGUIFUGDFJZAMUJANDONZCBUKCFEHPAUKBFE
+      DUKQGRZSTUHUCZUKBCFEHFEFUAABDEFGIUBUGUKBLMUMULUDUEUF $.
+  $}
+
+  ${
+    $d y z $.
+    $( ` th ` means ` [ y / x ] ph ` a la ~ dfsb1 . $)
+    dfsb1.p9 $e |- ( th <-> ( ( x = y -> ph ) /\ E. x ( x = y /\ ph ) ) ) $.
+
+    ${
+      sb7fALT.1 $e |- F/ z ph $.
+      $( Alternate version of ~ sb7f .  (Contributed by NM, 26-Jul-2006.)
+         (Revised by Mario Carneiro, 6-Oct-2016.)
+         (Proof modification is discouraged.)  (New usage is discouraged.) $)
+      sb7fALT $p |- ( th <-> E. z ( z = y /\ E. x ( x = z /\ ph ) ) ) $=
+        ( weq wi wa wex biid sb5fALT sbbiiALT sbco2ALT sb5ALT2 3bitr3i ) EDHZCE
+        HZAISAJCKZJZIRUAJEKJZRTIRTJEKZJZBUCUATUBUDEDUBLZUDLZAUACEUALGMNABUBCDEF
+        UEGOTUDEDUFPQ $.
+    $}
+
+    $d ph z $.
+    $( Alternate version of ~ dfsb7 .  (Contributed by NM, 28-Jan-2004.)
+       (Proof modification is discouraged.)  (New usage is discouraged.) $)
+    dfsb7ALT $p |- ( th <-> E. z ( z = y /\ E. x ( x = z /\ ph ) ) ) $=
+      ( nfv sb7fALT ) ABCDEFAEGH $.
+  $}
+
+  ${
+    $d z x $.  $d z y $.  $d z ph $.
+    $( Alternate definition of substitution.  Remark 9.1 in [Megill] p. 447 (p.
+       15 of the preprint).  This definition avoids disjoint variable provisos
```

Comment:
Maybe add the sentences: "This was the original definition before ~ df-sb. Note that it does not require dummy variables."